.github/workflows/build-test.yml

```diff
@@ -82,6 +82,7 @@ jobs:
           repository: aya-rs/aya
           path: aya
           submodules: recursive
+          ref: logs-wtf
```

Comment:
I assume this is to be removed when https://github.com/aya-rs/aya/pull/683 merges, yeah?